### DISCUSSION TITLE
Synchronization of postal addresses

### DISCUSSION THREAD

**Discussion by WebworkrNet** [FossifyOrg/General-Discussion#657] [General]
Regardless of Fossify and its predecessor, I have observed that there are different procedures for storing the snail mail address. Either the fields for street etc. are written in a common multi-line field or individual fields such as city and postal code are filled individually. The Samsung app continuously saves these locally in the block.
Simple Contacts used to do this too and Fossify has probably adopted this process. I haven't tested the case currently. The problem with Simple Mobile was that all contact information that belongs to the physical address was written into the street field on the server after synchronization via CardDAV. This always led to additional work due to necessary post-processing. Samsung, however, transmits this in a separate “block field”. The information is then not assigned to each individual data field, but there is also no incorrect assignment to the "street" field. When I pointed this out at the time, I received an answer from the former maintainer that nothing would change in current practice.

**Comment by Aga-C** [comment]
There's a pull request for changing how addresses are kept: https://github.com/FossifyOrg/Contacts/pull/57. You can take a look there to check if it's done properly.

**Comment by WebworkrNet** [comment]
It looks like this pull request and the feature requests #30 could be exactly what I'm looking for. Thank you for the hint, I had already looked through the tickets before, even subscribed to some of them for the sake of interest, but did not associate them with my scenario. 

I'll keep an eye on this development.
